Missing cstdint in utilfuncs

Compilation error on Linux Endeavour.
```
In file included from /home/slawomir/compr/compressonator/applications/_plugins/common/utilfuncs.h:40,
                 from /home/slawomir/compr/compressonator/applications/_plugins/common/utilfuncs.cpp:27:
/home/slawomir/compr/compressonator/applications/_plugins/common/cmp_fileio.h:44:1: error: ‘uintmax_t’ does not name a type
   44 | uintmax_t CMP_GetFileSize(const std::string& fileName);
      | ^~~~~~~~~
/home/slawomir/compr/compressonator/applications/_plugins/common/utilfuncs.h:49:19: error: ‘uint16_t’ was not declared in this scope
   49 | float HalfToFloat(uint16_t h);
      |                   ^~~~~~~~
/home/slawomir/compr/compressonator/applications/_plugins/common/utilfuncs.h:41:1: note: ‘uint16_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?
   40 | #include "cmp_fileio.h"
  +++ |+#include <cstdint>
   41 | 
/home/slawomir/compr/compressonator/applications/_plugins/common/utilfuncs.cpp:60:7: error: redefinition of ‘float HalfToFloat’
   60 | float HalfToFloat(uint16_t h) {
      |       ^~~~~~~~~~~
/home/slawomir/compr/compressonator/applications/_plugins/common/utilfuncs.h:49:7: note: ‘float HalfToFloat’ previously defined here
   49 | float HalfToFloat(uint16_t h);
      |       ^~~~~~~~~~~
/home/slawomir/compr/compressonator/applications/_plugins/common/utilfuncs.cpp:60:19: error: ‘uint16_t’ was not declared in this scope
   60 | float HalfToFloat(uint16_t h) {
      |                   ^~~~~~~~
/home/slawomir/compr/compressonator/applications/_plugins/common/utilfuncs.cpp:31:1: note: ‘uint16_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?
   30 | #include <assert.h>
  +++ |+#include <cstdint>
   31 | #define ASSERT assert
make[2]: *** [cmp_framework/CMakeFiles/CMP_Framework.dir/build.make:328: cmp_framework/CMakeFiles/CMP_Framework.dir/__/applications/_plugins/common/utilfuncs.cpp.o] Błąd 1
```

--- a/applications/_plugins/common/cmp_fileio.h
+++ b/applications/_plugins/common/cmp_fileio.h
@@ -20,6 +20,7 @@
 
 #include <string>
 #include <vector>
+#include <cstdint>
 
 enum CMP_PATHTYPES
 {

--- a/applications/_plugins/common/utilfuncs.h
+++ b/applications/_plugins/common/utilfuncs.h
@@ -36,6 +36,7 @@
 #include <iostream>
 HWND FindTopLevelWindow(TCHAR *pszName);
 #endif
+#include <cstdint>
 
 #include "cmp_fileio.h"
 

